examples/MQ/pixelDetector/src/PixelDigiBinSource.cxx

```diff
@@ -138,11 +143,12 @@ Bool_t PixelDigiBinSource::ActivateObject(TObject** obj, const char* BrName)
     return kTRUE;
 }
 
-void PixelDigiBinSource::Close() { fInputFile.close(); }
-
```

Comment:
Sadly enough this is IMHO also part of the public API: People can call Close and expect the resource to be closed. Of course they now get a new warning, that they should not call Close.

One could argue, that we just "leak" the resource until the dtor is called (hopefully) anyway…

I guess, I would appreciate it, if we keep this for a bit until we start removing things?

Reply:
Okay, please excuse! My fault!

In `examples/` we can of course perform everything we want (we even should), because examples are not public API. They show, how one should use things.

So really, here everything you did, was good!

My fault!